Roys/app/src/main/java/com/dwg_karrier/roys/ScriptedData.java

```diff
@@ -0,0 +1,23 @@
+package com.dwg_karrier.roys;
```

Comment:
I think this file is added on so many PRs. Would you please check the master and include the necessary files only?